### PR TITLE
stomp: use rabbitmq

### PR DESCRIPTION
**Pull request by hdonnay**
This updates the `stomp` package and the tests to use only RabbitMQ instead of both RabbitMQ and ActiveMQ.

### CODE REVIEW

**Comment by hdonnay** [comment]
@RTann, I'm very sorry for forgetting to put this in draft :(

File: local-dev/traefik/config/rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+http:
+  routers:
+    activemq:
```
Comment (crozzy) [inline local-dev/traefik/config/rabbitmq.yaml]: should this be `rabbitmq`?

**Review by crozzy — COMMENTED** [review]
Looks good, just one comment

File: local-dev/traefik/config/rabbitmq.yaml (hunk 1 above)
Comment (hdonnay) [reply local-dev/traefik/config/rabbitmq.yaml]: yes

**Review by crozzy — APPROVED** [review]

**Pull request merged**